modules/renter/hostdb/hostweight.go

```diff
@@ -47,10 +44,7 @@ var (
 
 	// priceExponentiation is the number of times that the weight is divided by
 	// the price.
-	//
-	// NOTE: Changing this value upwards means that the baseWeight will need to
-	// be increased.
-	priceExponentiation = 4
+	priceExponentiation = 5
```

Comment:
I didn't want to do this because it makes certain attacks easier, but we've taken steps even then to limit them as much as possible. More importantly though, we don't have nearly enough hosts, and without the filter being this sharp some expensive ones can still make the top-50.